packages/cli/src/e2e.test.ts

```diff
@@ -0,0 +1,179 @@
+import {
+    afterAll,
+    beforeAll,
+    beforeEach,
+    describe,
+    expect,
+    test,
+    vitest,
+} from 'vitest';
+import * as prettier from 'prettier';
+import fs from 'fs';
+import path from 'path';
+import { configFileSchema } from './validators/schemas';
+import { cli } from './cli';
+import { randomUUID } from 'crypto';
+
+describe('e2e', () => {
```

Comment:
Conceptually, what's the difference between `src/e2e.test.ts` and `src/cli.test.ts`?

Reply:
`e2e.test.ts` is just where I'm adding any tests where we're testing the results (and not just the `CLI` function, like the tests in `cli.test.ts`) and/or are storing the side-effects, using it the same way you used it in the introspect package